R/context.R

```diff
@@ -21,3 +21,26 @@ context <- function(desc) {
 end_context <- function() {
   get_reporter()$.end_context()
 }
+
+
+#' @describeIn context Simplify a test filename to a context name.
+#'
+#' @param filename A character string corresponding to a
```

Comment:
I'd say: "A character string giving the filename of a test file. Standard prefix (`test-`) and suffix (`.R`) are stripped"